mix.exs

```diff
@@ -31,11 +31,11 @@ defmodule NimbleTemplate.MixProject do
   defp deps do
     [
       {:phoenix, "~> 1.5.7"},
-      {:jason, "~> 1.2.2"},
-      {:httpoison, "~> 1.7.0"},
-      {:mimic, "~> 1.3.1", only: :test},
-      {:credo, "~> 1.4.0", only: [:dev, :test], runtime: false},
-      {:ex_doc, "~> 0.23.0", only: :dev, runtime: false}
+      {:jason, "~> 1.2"},
+      {:httpoison, "~> 1.7"},
+      {:mimic, "~> 1.3", only: :test},
+      {:credo, "~> 1.4", only: [:dev, :test], runtime: false},
+      {:ex_doc, "~> 0.23", only: :dev, runtime: false}
```

Comment:
hi @andyduong1920 !

Should we arrange these dependencies by alphabet?

Reply:
Sure @byhbt :D

Reply:
btw, i see this issue happened many times, should we make it as convention? :)

Reply:
Agree, we should add it to our convention @byhbt 👍

Reply:
Fixed it in https://github.com/nimblehq/elixir-templates/pull/97/commits/f99c162050128c1356275faa603ffa0906746c4a

Reply:
Could you open an issue on our compass to adding this issue to our Elixir convention @byhbt 🤔

Reply:
Sure thing, I will do that today. Thanks

Reply:
Thanks @byhbt